### ISSUE TITLE
Crash when trying to save profile with AD cert info

### ISSUE THREAD

**Issue by ygini** [ProfileCreator/ProfileCreator#20]
Where is the console log with crash inside.

```
24/06/2016 14:14:27,578 [VERBOSE] -[PFCProfileEditor buttonSave:]
24/06/2016 14:14:27,578 [DEBUG] Saving settings for manifest domain: com.apple.ADCertificate.managed.manifest
24/06/2016 14:14:27,579 [DEBUG] Settings to save: {
    "0876B806-BF31-4EA5-86E3-15657DCA7420" =     {
        Value = "srv-cert";
    };
    "142A27DF-6AD5-44F7-B1B2-6735C52D171C" =     {
        Value = "cert ca";
    };
    "22A6513F-65DD-48C8-AB89-D7FCDC9272FA" =     {
        Value = "AD Cert desc";
    };
    "3C9F3FFE-CC74-464F-A87F-52F2012AC77A" =     {
        Value = Ordinateur;
    };
    "55EF75E2-3BB6-48B4-A0F2-35F0945E4087" =     {
        Value = user;
    };
    "602DAB08-AE68-4536-B31C-8425CE23FF61" =     {
        ValueCheckbox = 1;
    };
    "7F59E5CA-AFA9-4E92-B393-344A6EB1D922" =     {
        ValueCheckbox = 1;
    };
    "DD667296-F6F6-4B0D-9ADC-D3E80254C915" =     {
        Value = RPC;
    };
    "DE282055-F4AE-4F9E-A21E-EC12ED7E9623" =     {
        Value = pass;
    };
}
24/06/2016 14:14:27,579 [DEBUG] Current manifest settings count: 1
24/06/2016 14:14:27,579 [DEBUG] Current manifest tab count: 1
24/06/2016 14:14:27,579 [DEBUG] Replacing current settings at index 0
24/06/2016 14:14:27,579 [VERBOSE] Saving settings: (
        {
        "0876B806-BF31-4EA5-86E3-15657DCA7420" =         {
            Value = "srv-cert";
        };
        "142A27DF-6AD5-44F7-B1B2-6735C52D171C" =         {
            Value = "cert ca";
        };
        "22A6513F-65DD-48C8-AB89-D7FCDC9272FA" =         {
            Value = "AD Cert desc";
        };
        "3C9F3FFE-CC74-464F-A87F-52F2012AC77A" =         {
            Value = Ordinateur;
        };
        "55EF75E2-3BB6-48B4-A0F2-35F0945E4087" =         {
            Value = user;
        };
        "602DAB08-AE68-4536-B31C-8425CE23FF61" =         {
            ValueCheckbox = 1;
        };
        "7F59E5CA-AFA9-4E92-B393-344A6EB1D922" =         {
            ValueCheckbox = 1;
        };
        "DD667296-F6F6-4B0D-9ADC-D3E80254C915" =         {
            Value = RPC;
        };
        "DE282055-F4AE-4F9E-A21E-EC12ED7E9623" =         {
            Value = pass;
        };
    }
)
24/06/2016 14:14:27,579 [DEBUG] Profile runtime path: /Users/ygi/Library/Application Support/ProfileCreator/Profiles/ABFEA23A-5612-4A8B-B0BB-49C264164C4D.pfcconf
24/06/2016 14:14:27,579 [DEBUG] Profile save path: /Users/ygi/Library/Application Support/ProfileCreator/Profiles/ABFEA23A-5612-4A8B-B0BB-49C264164C4D.pfcconf
2016-06-24 14:14:27.762 ProfileCreator[26052:1466346] -[__NSCFString allKeys]: unrecognized selector sent to instance 0x600000c5b210
2016-06-24 14:14:27.762 ProfileCreator[26052:1466346] -[__NSCFString allKeys]: unrecognized selector sent to instance 0x600000c5b210
2016-06-24 14:14:27.763 ProfileCreator[26052:1466346] (
	0   CoreFoundation                      0x00007fff90ff14f2 __exceptionPreprocess + 178
	1   libobjc.A.dylib                     0x00007fff84e9cf7e objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff9105b1ad -[NSObject(NSObject) doesNotRecognizeSelector:] + 205
	3   CoreFoundation                      0x00007fff90f61571 ___forwarding___ + 1009
	4   CoreFoundation                      0x00007fff90f610f8 _CF_forwarding_prep_0 + 120
	5   ProfileCreator                      0x00000001000873ab -[PFCManifestParser tableViewColumnsFromArray:settings:] + 2971
	6   ProfileCreator                      0x0000000100085d81 -[PFCManifestParser manifestContentFromDict:settingsDict:] + 1633
	7   ProfileCreator                      0x0000000100084749 -[PFCManifestParser manifestFromPlistAtURL:settings:] + 425
	8   ProfileCreator                      0x0000000100095ff4 -[PFCManifestLibrary manifestsFromPlistsAtURL:library:] + 708
	9   ProfileCreator                      0x0000000100093f21 -[PFCManifestLibrary libraryUserLibraryPreferencesLocal:acceptCached:] + 385
	10  ProfileCreator                      0x0000000100092e44 -[PFCManifestLibrary libraryAll:acceptCached:] + 276
	11  ProfileCreator                      0x00000001000949c4 -[PFCManifestLibrary manifestsWithDomains:] + 132
	12  ProfileCreator                      0x000000010000b729 -[PFCProfileEditor saveProfile:] + 2889
	13  ProfileCreator                      0x000000010000e0ec -[PFCProfileEditor buttonSave:] + 2012
	14  libsystem_trace.dylib               0x00007fff8eb4d07a _os_activity_initiate + 75
	15  AppKit                              0x00007fff946dadbd -[NSApplication sendAction:to:from:] + 460
	16  AppKit                              0x00007fff946ecf12 -[NSControl sendAction:to:] + 86
	17  AppKit                              0x00007fff946ece3c __26-[NSCell _sendActionFrom:]_block_invoke + 131
	18  libsystem_trace.dylib               0x00007fff8eb4d07a _os_activity_initiate + 75
	19  AppKit                              0x00007fff946ecd99 -[NSCell _sendActionFrom:] + 144
	20  libsystem_trace.dylib               0x00007fff8eb4d07a _os_activity_initiate + 75
	21  AppKit                              0x00007fff946eb3be -[NSCell trackMouse:inRect:ofView:untilMouseUp:] + 2693
	22  AppKit                              0x00007fff94733f04 -[NSButtonCell trackMouse:inRect:ofView:untilMouseUp:] + 744
	23  AppKit                              0x00007fff946e9ae8 -[NSControl mouseDown:] + 669
	24  AppKit                              0x00007fff94c3e3c9 -[NSWindow _handleMouseDownEvent:isDelayedEvent:] + 6322
	25  AppKit                              0x00007fff94c3f3ad -[NSWindow _reallySendEvent:isDelayedEvent:] + 212
	26  AppKit                              0x00007fff9467e539 -[NSWindow sendEvent:] + 517
	27  AppKit                              0x00007fff945fea38 -[NSApplication sendEvent:] + 2540
	28  AppKit                              0x00007fff94465df2 -[NSApplication run] + 796
	29  AppKit                              0x00007fff9442f368 NSApplicationMain + 1176
	30  ProfileCreator                      0x0000000100033492 main + 34
	31  libdyld.dylib                       0x00007fff9532d5ad start + 1
	32  ???                                 0x0000000000000003 0x0 + 3
)
```

**Comment by erikberglund** [comment]
This is expected as the exporting is currently being worked through. I've added the UI manifest parts for all os x payloads, but the payload keys and the exporting is not complete. I will update this issue: https://github.com/ProfileCreator/ProfileCreator/issues/16 when I've confirmed and tested each manifest/payload.

I did a refactor of all cell types so the export has not been updated to match that (yet). The only tested payload currently is "Identification". But, as it's not per payload but per cell type, that will mean that as soon as I've worked though all types it will work for all payloads.
